broker/lib/fabrik/VirtualHostRelationMapper.js

```diff
@@ -2,9 +2,9 @@
 
 const Promise = require('bluebird');
 const logger = require('../logger');
-const bosh = require('../bosh');
+const bosh = require('../../../data-access-layer/bosh');
```

Comment:
Can we directly name the sub package directory **data-access-layer** as **dal**?

Reply:
It does not sound obvious, h nice the full name. It’s not that big hence I thought we can use it.